MetaMask/metamask-extension
# UX: Multichain: Remove bottom navigation

## **Description**

Removes the abandoned `AppFooter`, `SelectActionModal`, and the UI around it.

## **Related issues**

Fixes:  https://github.com/MetaMask/MetaMask-planning/issues/1824

## **Manual testing steps**

1. `MULTICHAIN=1 yarn start`
2. No longer see the bottom navigation, do see the EthOverview buttons restored.

## **Screenshots/Recordings**



### **Before**

N/A

### **After**

<img width="848" alt="SCR-20240122-lify" src="https://github.com/MetaMask/metamask-extension/assets/46655/3f0984cc-e5d9-4581-a56b-76d51d04ed3c">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
